api/test/h3-data/h3-risk-map.spec.ts

```diff
@@ -269,51 +386,83 @@ describe('H3 Data Module (e2e) - Risk map', () => {
         fakeTable,
         fakeColumn,
       });
+
+    const indicatorH3Data = await h3DataMock(
+      fakeTable2,
+      fakeColumn2,
+      h3AlternativeIndicatorFixture,
+      indicator.id,
+    );
     jest.spyOn(h3DataRepository, 'getCarbonEmissionsRiskMapByResolution');
 
-    const response = await request(app.getHttpServer())
+    const responseRes6 = await request(app.getHttpServer())
       .get(`/api/v1/h3/map/risk`)
       .query({
         indicatorId: indicator.id,
         year: 2020,
         materialId: material.id,
         resolution: 6,
       });
+
+    const responseRes3 = await request(app.getHttpServer())
+      .get(`/api/v1/h3/map/risk`)
+      .query({
+        indicatorId: indicator.id,
+        year: 2020,
+        materialId: material.id,
+        resolution: 3,
+      });
+
     expect(
       h3DataRepository.getCarbonEmissionsRiskMapByResolution,
     ).toHaveBeenCalledWith(
-      h3Data,
+      indicatorH3Data,
       h3Data,
       h3Data,
       deforestationH3Data,
       unitConversion.factor,
       6,
     );
-
-    expect(response.body.data).toEqual([
-      {
-        h: '861203a4fffffff',
-        v: 1000000,
-      },
-    ]);
-    expect(response.body.metadata).toEqual({
-      quantiles: [
-        1000000, 1000000, 1000000, 1000000, 1000000, 1000000, 1000000,
-      ],
-      unit: 'tonnes',
-    });
+    expect(
+      h3DataRepository.getCarbonEmissionsRiskMapByResolution,
+    ).toHaveBeenCalledWith(
+      indicatorH3Data,
+      h3Data,
+      h3Data,
+      deforestationH3Data,
+      unitConversion.factor,
+      3,
+    );
+    expect(responseRes6.body.data).toEqual(
+      expect.arrayContaining(expectedCarbonEmissionsRes6Data),
+    );
+    expect(responseRes6.body.metadata).toEqual(
+      expectedCarbonEmissionsRes6Metadata,
+    );
+    expect(responseRes3.body.data).toEqual(
+      expect.arrayContaining(expectedCarbonEmissionsRes3Data),
+    );
+    expect(responseRes3.body.metadata).toEqual(
+      expectedCarbonEmissionsRes3Metadata,
+    );
   });
   // TODO: Update assertion as soon as actual calculus is validated
```

Comment:
@yulia-bel @alexeh is this still relevant?

Reply:
I guess **NOW** it's the moment to remove it! :)